helm_deploy/apply-for-legal-aid/Chart.yaml

```diff
@@ -15,15 +17,19 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.2.0
+version: 0.3.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: 1.17.0
+appVersion: 1.18.0
 
 dependencies:
   - name: postgresql
     version: ^11.0.0
     condition: postgresql.enabled
     repository: https://charts.bitnami.com/bitnami
+  - name: redis
+    version: ^16.0.0 # redis 6.x.x
```

Comment:
Presumably this will be almost immediately switched to 7 with the sidekiq update?

Reply:
yeah, i need to mention that in the new "part 2" ticket actually https://dsdmoj.atlassian.net/browse/AP-4556